server/sonar-server/src/main/java/org/sonar/server/user/DefaultUserService.java

```diff
@@ -39,11 +39,13 @@
 public class DefaultUserService implements RubyUserService {
 
   private final UserService userService;
+  private final UserUpdater userUpdater;
   private final UserFinder finder;
   private final UserDao dao;
 
-  public DefaultUserService(UserService userService, UserFinder finder, UserDao dao) {
+  public DefaultUserService(UserService userService, UserUpdater userUpdater, UserFinder finder, UserDao dao) {
```

Comment:
You could replace UserServer by UserIndex, it will allow to completely remove UserService !